app/views/research_sessions/questions/topic.html.erb

```diff
@@ -3,23 +3,45 @@
   <%= render 'shared/error_summary' %>
   <h1 class="subtitle-large">Topic</h1>
   <%= barnardos_form_with model: @research_session, url: wizard_path do |form| %>
-    <%= form.labelled_text_field(
-          :topic,
-          text_options: {
-            placeholder: t('helpers.placeholder.research_session.topic'),
-            autofocus: true
-          }
-        )
-    %>
+    <div class="reactive-container">
+      <div class="reactive-container__form">
+        <%= form.labelled_text_field(
+              :topic,
+              text_options: {
+                placeholder: t('helpers.placeholder.research_session.topic'),
+                autofocus: true,
+                'data-previewed-by' => 'TopicPurposePreviews'
+              }
+            )
+        %>
 
-    <%= form.labelled_text_field(
-          :purpose,
-          text_options: {
-            placeholder: t('helpers.placeholder.research_session.purpose'),
-            autofocus: true
-          }
-        )
-    %>
+        <%= form.labelled_text_field(
+              :purpose,
+              text_options: {
+                placeholder: t('helpers.placeholder.research_session.purpose'),
+                autofocus: true,
+                'data-previewed-by' => 'TopicPurposePreviews'
+              }
+            )
+        %>
+      </div>
+
+      <div class="reactive-container__preview">
+        <section class="reactive-preview__section">
+          <%=
+            react_component(
+              'TopicPurposePreviews',
+              current_step_params.merge(
+                labels: {
+                  topic: t('research_session_attr_labels.topic'),
+                  purpose: t('research_session_attr_labels.purpose')
```

Comment:
This approach – to do translation manually and throw things over the wall – will work here but would not be sustainable for larger volumes of stuff. Suggest a framework such as [i18next](https://github.com/i18next/react-i18next) or Yahoo!'s [react-intl](https://github.com/yahoo/react-intl) along with an adapter for the `en.yml` file to load everything up onec and have the same semantics in both Ruby and JS worlds